libraries/AP_InertialSensor/examples/INS_generic/INS_generic.cpp

```diff
@@ -23,6 +23,12 @@ void setup(void)
 
     // display initial values
     display_offsets_and_scaling();
+
+    // display number of detected accels/gyros
+    hal.console->printf("\n");
+    hal.console->printf("Number of detected accels : %d\n",ins.get_accel_count());
+    hal.console->printf("Number of detected gyros  : %d\n\n",ins.get_gyro_count());
```

Comment:
We should use `printf("... %" PRIu8 "...", ...)` or at least `%u`. Also, missing space after the coma :-)